lib/Controller/ChatController.php

```diff
@@ -689,37 +692,58 @@ public function deleteMessage(int $messageId): DataResponse {
 	#[RequireModeratorOrNoLobby]
 	#[RequireLoggedInParticipant]
 	#[UserRateLimit(limit: 60, period: 3600)]
-	public function remindLater(int $messageId, int $timestamp): DataResponse {
+	public function setReminder(int $messageId, int $timestamp): DataResponse {
 		try {
-			$message = $this->chatManager->getComment($this->room, (string) $messageId);
+			$this->chatManager->getComment($this->room, (string) $messageId);
 		} catch (NotFoundException) {
 			return new DataResponse([], Http::STATUS_NOT_FOUND);
 		}
 
-		$this->chatManager->queueRemindLaterBackgroundJob(
-			$this->room,
-			$message,
-			$this->participant->getAttendee(),
+		$reminder = $this->reminderService->setReminder(
+			$this->participant->getAttendee()->getActorId(),
+			$this->room->getToken(),
+			$messageId,
 			$timestamp
 		);
 
-		return new DataResponse([], Http::STATUS_CREATED);
+		return new DataResponse($reminder->jsonSerialize(), Http::STATUS_CREATED);
 	}
 
 	#[NoAdminRequired]
 	#[RequireModeratorOrNoLobby]
 	#[RequireLoggedInParticipant]
-	public function dismissReminder(int $messageId): DataResponse {
+	public function getReminder(int $messageId): DataResponse {
 		try {
-			$message = $this->chatManager->getComment($this->room, (string) $messageId);
+			$this->chatManager->getComment($this->room, (string) $messageId);
 		} catch (NotFoundException) {
 			return new DataResponse([], Http::STATUS_NOT_FOUND);
 		}
```

Comment:
Do we need this part?
If the reminder exists, message should exists too, right?